### PR TITLE
Added Crewsimov and set to default - CONTROVERSIAL

### PR DESCRIPTION
**Pull request by JamieD1**
Hear me out here,

Asimov has several flaws in a logical space station which has different species on,
For one the AI has to obey all syndicate nuke ops because they are human, why would NT let this happen?
Also Lizards (I Know) should at least be protected from the ai.

Also this prevents borgs and the ai going after certain antags like Changelings and Vampires as they will still be considered crew unless removed by command.

This also saves on a lot of admin time, no more is it human is it not, simple solution, if they are on the crew manifest then bad, if they are not on the manifest then ai and borgs can actually do shit instead of watching themselves get blown up by nukies.

Obviously this will cause alot of debate and im shit at explaining so feel free to fight for or against this, just raising it as a concern that would in my opinion make the server a better place to be for silicon and non silicon players.
 
#### Changelog

:cl:  
tweak: Added Crewsimov and set to default lawset 
/:cl:

### CODE REVIEW

**Comment by Oakboscage** [comment]
t. seething lizard player

**Comment by athena413** [comment]
yes

**Comment by Colburn3000** [comment]
I would honestly be fine with Crewsimov, but it's not a hill I am going to die on.

**Comment by GoreDem** [comment]
I sure do wonder how long it will take for people to argue a shit ton about this subject

**Comment by JamieD1** [comment]
Ideally i'd like a discussion to understand why people disapprove of it.

**Comment by GoreDem** [comment]
change man bad re

**Comment by yacabo** [comment]
Asimov is supposed to be extremely flawed. That's the point.

**Comment by swissloaf** [comment]
> Asimov is supposed to be extremely flawed. That's the point.

crewsimov is flawed too. traitors are still crew unless theres a bypass law for that, just as traitors can also be human

**Comment by JamieD1** [comment]
> Asimov is supposed to be extremely flawed. That's the point.

So if everyone knows it has flaws why would people not change it, seems counterproductive

**Comment by Mr-Space** [comment]
It's a feature not a bug

**Comment by HomocidlePancakes** [comment]
Hello, the quarantine has me bored out of my balls, so I’ve crawled from under my rock to explain to you why this is fucking stupid. Let’s tear down the PR body one by one:

>Asimov has several flaws in a logical space station which has different species on,
For one the AI has to obey all syndicate nuke ops because they are human, why would NT let this happen?

I am about to do a revelation to you, dear op. It may be too much for your virgin sethtide asshole, so please, clench it tightly and brace yourself.

**You are not on a logical space station**

You are probably gasping for air right now, tears in your eyes, but that’s the truth - you have literal wizards running around along with other illogical shit. I could probably write a 500 essay just on this point alone, but I won’t.

Now, you may be asking: “P-pancakes senpai, bu-but why is it like that”. Well, my young padavan, it’s very simple - AI is a fucking all-seeing and all-knowing cunt and people play it like a cunt. It’s cancer to deal with, but thankfully, asimov exists to counteract it to a degree. 

>Also Lizards (I Know) should at least be protected from the ai.

Um, why? Want to be protected by the AI? Play a human. It’s that simple. When people pick a non-human, they have to understand it comes with some downsides, one which is AI. You shoot yourself in the foot, go “not my fault lol” and then make this PR. Fuck off.

> Also this prevents borgs and the ai going after certain antags like Changelings and Vampires as they will still be considered crew unless removed by command.

Gross misunderstanding of AI laws right here. Changelings and vampires are shapeshifters and it has been always been accepted that they have disguised themself as a crewmember, while the actual lad is dead. 

> This also saves on a lot of admin time, no more is it human is it not

Did you ran out of points while thinking up arguments on this PR? It’s a gameplay change first, admin second. 

>simple solution, if they are on the crew manifest then bad, if they are not on the manifest then ai and borgs can actually do shit instead of watching themselves get blown up by nukies.

Sorry, but I fail to see how identifying humans from non-humans is so mind-numbingly hard that you have to simplify it. If you are struggling with something so simple as this, then you either have a dent on your skull, or you haven’t even bothered figuring it out. Also, good chance you ran out of points and just tried to make some bloat. 

>if they are not on the manifest then ai and borgs can actually do shit instead of watching themselves get blown up by nukies.

Again, gross misunderstanding of silicons in general. I shit you not, dear op, but you have plenty of things to do as a silicon during nuke ops. You can support crew by bringing them supplies, fixing breaches that will eventually pop-up, dragging injured crewmembers out of the heat or act as a bullet sponge if the situation warrants it. If you think that as a silicon, the only response you have is “i musd valid ALL DA ANTEG”, then you should be banned from silicons this fucking instant. Follow your laws, fucker, or don’t play AI.

tl;dr OP is newf... newfriend and has no idea what he is doing or talking about, give him a perma from silicons. That is all.

**Comment by JamieD1** [comment]
> Hello, the quarantine has me bored out of my balls, so I’ve crawled from under my rock to explain to you why this is fucking stupid. Let’s tear down the PR body one by one:
> 
> > Asimov has several flaws in a logical space station which has different species on,
> > For one the AI has to obey all syndicate nuke ops because they are human, why would NT let this happen?
> 
> I am about to do a revelation to you, dear op. It may be too much for your virgin sethtide asshole, so please, clench it tightly and brace yourself.
> 
> **You are not on a logical space station**
> 
> You are probably gasping for air right now, tears in your eyes, but that’s the truth - you have literal wizards running around along with other illogical shit. I could probably write a 500 essay just on this point alone, but I won’t.
> 
> Now, you may be asking: “P-pancakes senpai, bu-but why is it like that”. Well, my young padavan, it’s very simple - AI is a fucking all-seeing and all-knowing cunt and people play it like a cunt. It’s cancer to deal with, but thankfully, asimov exists to counteract it to a degree.
> 
> > Also Lizards (I Know) should at least be protected from the ai.
> 
> Um, why? Want to be protected by the AI? Play a human. It’s that simple. When people pick a non-human, they have to understand it comes with some downsides, one which is AI. You shoot yourself in the foot, go “not my fault lol” and then make this PR. Fuck off.
> 
> > Also this prevents borgs and the ai going after certain antags like Changelings and Vampires as they will still be considered crew unless removed by command.
> 
> Gross misunderstanding of AI laws right here. Changelings and vampires are shapeshifters and it has been always been accepted that they have disguised themself as a crewmember, while the actual lad is dead.
> 
> > This also saves on a lot of admin time, no more is it human is it not
> 
> Did you ran out of points while thinking up arguments on this PR? It’s a gameplay change first, admin second.
> 
> > simple solution, if they are on the crew manifest then bad, if they are not on the manifest then ai and borgs can actually do shit instead of watching themselves get blown up by nukies.
> 
> Sorry, but I fail to see how identifying humans from non-humans is so mind-numbingly hard that you have to simplify it. If you are struggling with something so simple as this, then you either have a dent on your skull, or you haven’t even bothered figuring it out. Also, good chance you ran out of points and just tried to make some bloat.
> 
> > if they are not on the manifest then ai and borgs can actually do shit instead of watching themselves get blown up by nukies.
> 
> Again, gross misunderstanding of silicons in general. I shit you not, dear op, but you have plenty of things to do as a silicon during nuke ops. You can support crew by bringing them supplies, fixing breaches that will eventually pop-up, dragging injured crewmembers out of the heat or act as a bullet sponge if the situation warrants it. If you think that as a silicon, the only response you have is “i musd valid ALL DA ANTEG”, then you should be banned from silicons this fucking instant. Follow your laws, fucker, or don’t play AI.
> 
> tl;dr OP is newf... newfriend and has no idea what he is doing or talking about, give him a perma from silicons. That is all.

Dude I dont give a fuck, I made this to start a discussion about the whole thing, I dont give a shit if its added or not, I am just making the opportunity available to people. Dont know why you have to be such an arsehole.

**Comment by HomocidlePancakes** [comment]
I spy with my little eye... A contradiction.

>Dude I dont give a fuck

> I made this to start a discussion about the whole thing

Well, shit, my post may have some exotic language, but it does address a bunch of points too. If you don’t have the patience to actually think and instead call anyone who disagrees with you an asshole, then you shouldn’t have brought up this topic in the first place. I’ll take pity on you and rewrite the damn thing without anything that may offend you, but it’s a one time offer. If you just keep responding like that, I will go in dry and full throttle on you.

>Asimov has several flaws in a logical space station which has different species on,
For one the AI has to obey all syndicate nuke ops because they are human, why would NT let this happen?

You are not on a logical space station

AI starting with a flawed lawset was an entirely conscious decision. AI players are horrible to deal with for anyone on the receiving end. It’a an omnipotent machine that acts with no warning. Thankfully, asimov was made to counteract that to some degree.

>Also Lizards (I Know) should at least be protected from the ai.

Anyone who picks a non-human has to understand it comes with some flaws, one of which is AI. If you don’t like that, don’t play xenos. It’s a choice, that’s all there is to it.

>Also this prevents borgs and the ai going after certain antags like Changelings and Vampires as they will still be considered crew unless removed by command.

Gross misunderstanding of AI laws right here. Changelings and vampires are shapeshifters and it has been always been accepted that they have disguised themself as a crewmember, while the actual lad is dead.

>This also saves on a lot of admin time, no more is it human is it not

Gameplay change first, admin change second. Please, stay within the scope of the topic.

>simple solution, if they are on the crew manifest then bad, if they are not on the manifest then ai and borgs can actually do shit instead of watching themselves get blown up by nukies.

Sorry, but I fail to see how identifying humans from non-humans is so hard that you have to simplify it. You may present it as a problem, but it’s not. 

>if they are not on the manifest then ai and borgs can actually do shit instead of watching themselves get blown up by nukies.

Again, gross misunderstanding of silicons in general. You have plenty of things to do as a silicon during nuke ops. You can support crew by bringing them supplies, fixing breaches that will eventually pop-up, dragging injured crewmembers out of the heat or act as a bullet sponge if the situation warrants it. Silicons are harm-preventing machines, not cops. Follow your laws.

**Comment by MegaEmpirical** [comment]
Asimov is meant to be flawed.

**Comment by zxmongoose** [comment]
I'm gonna touch on the "saves a lot of admin time" bit because I think Pancakes has summed up a lot of things quite well. 

This wouldn't save any admin time, and Asimov is actually one of the easier lawsets to admin. Asimov has comparatively few variables to balance when it comes to determining whether an AI's actions were valid given certain IC circumstances. Most other lawsets require you to really dig into the situation, and gather multiple pers pectives, so the admin can determine if the action had reasonable logic to its. Asimov boils down to: was the person harmed human? And if not, was there a valid reason for the AI to do what harm it did?

Finally, as others have said, the drawbacks behind Asimov are exactly the reason it's used as the "base" lawset. Every nonhuman species has extra benefits, as well as some drawbacks, that humans don't get. Humans are incredibly basic, with their only bits of uniqueness coming from their ability to sign up for command positions and their higher level of importance to Asimov silicons.

**Comment by jcatjmeow** [comment]
I don't like this because on crewsimov, nukies, wizards, prisoner ghost roles and other such people not on the manifest aren't considered crew and can be instantly valided

**Comment by Bibby0110** [comment]
what an absolutely horrible idea
